tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -274,13 +274,20 @@ if(LINK_WITH_PYTHON)
   endif()
 endif()
 
-if(CMAKE_HOST_UNIX)
-  if(NOT APPLE)
-    list(APPEND test_dependencies dl)
-  endif()
+if(CMAKE_HOST_UNIX AND NOT APPLE)
+  list(APPEND test_dependencies dl)
 endif()
 
 # Executable name will be EnergyPlus_tests
 # Execute energyplus_tests --help for options using gtest runner
 # Execute energyplus_tests with no arguments to run all tests
 create_test_targets(energyplus "${test_src}" "${test_dependencies}" True)
+
+
+set(test_src
+    Fixtures/EnergyPlusFixture.cc
+    Fixtures/EnergyPlusFixture.hh
+    CommandLineInterface.unit.cc
+    main.cc
+)
+create_test_targets(energyplusapi "${test_src}" "${test_dependencies}" True)
```

Comment:
So we now have an energyplusapi_tests binary?  This is totally fine if so, and will be included when I run `ctest` either way, just wanting to confirm.

Reply:
Yes. I was considering renaming it "energyplus_cli_tests" or something, but then that means I can't use create_test_targets, so either I have to modify it + every point that calls it, or I have to duplicate the code in the macro. So I figured that was fine...